apps/admin/backend/src/server.ts

```diff
@@ -70,24 +51,38 @@ export async function start({
         'workspace path could not be determined; pass a workspace or run with ADMIN_WORKSPACE'
       );
     }
-    /* istanbul ignore next */
     resolvedWorkspace = createWorkspace(workspacePath);
   }
 
-  // clear any cached data
+  // Clear any cached data
   resolvedWorkspace.clearUploads();
 
-  const usb: Usb = { getUsbDrives };
-
+  let resolvedApp = app;
   /* istanbul ignore next */
-  const resolvedApp =
-    app ??
-    buildApp({
+  if (!resolvedApp) {
+    const auth = new DippedSmartCardAuth({
+      card:
+        isFeatureFlagEnabled(BooleanEnvironmentVariableName.USE_MOCK_CARDS) ||
+        isIntegrationTest()
+          ? new MockFileCard()
+          : new JavaCard(
+              constructJavaCardConfig({ includeCardProgrammingConfig: true })
+            ),
```

Comment:
The core change in this commit is the removal of `new MemoryCard(...)` from all the app backends.

But I also had to shuffle code around such that we don't initialize a real auth instance in tests. Initializing a real auth instance, even if it goes unused, causes tests to fail since we can't actually communicate with a card reader in the test environment.

Had to make similar changes to all the app backends.